mysql-test/main/pfs_functions.test

```diff
@@ -0,0 +1,108 @@
+--echo #
+--echo # Tests for the Performance Schema native function format_pico_time()
+--echo #
+--echo
+
+SELECT format_pico_time(NULL);
```

Comment:
Add in test case scientific notation for arguments that is missing and would be easier to understand:
```sql
+SELECT format_pico_time(3e9);
+format_pico_time(3e9)
+3.00 ms
+SELECT format_pico_time(4e6);
+format_pico_time(4e6)
+4.00 us
+SELECT format_pico_time(5e3);
+format_pico_time(5e3)
+5.00 ns
+SELECT format_pico_time(6e2);
+format_pico_time(6e2)
+600 ps
```